coordinator/tests/services/spine-communication.spec.ts

```diff
@@ -1,5 +1,5 @@
 import 'jest'
-import moxios from 'moxios'
```

Comment:
This line shows as an error in intellij: 
`TS1259: Module '"/home/conner/NHSdir/electronic-prescription-service-api/coordinator/node_modules/@types/moxios/index"' can only be default-imported using the 'esModuleInterop' flag 

index.d.ts(194, 1): This module is declared with using 'export =', and can only be used with a default import when using the 'esModuleInterop' flag.`

Reply:
Is anyone else seeing this problem?